public/scripts/loader.js

```diff
@@ -27,24 +27,36 @@ export async function hideLoader() {
     }
 
     return new Promise((resolve) => {
-        // Spinner blurs/fades out
-        $('#load-spinner').on('transitionend webkitTransitionEnd oTransitionEnd MSTransitionEnd', function () {
+        const spinner = $('#load-spinner');
+        
+        // Apply the styles
+        spinner.css({
+            'filter': 'blur(15px)',
+            'opacity': '0',
+        });
+
+        // Check if transitions are disabled by comparing computed style
+        const transitionDuration = getComputedStyle(spinner[0]).transitionDuration;
+        const hasTransitions = transitionDuration !== '0s' && transitionDuration !== '0';
```

Comment:
This will not catch ms or other units.

Conside the following
```suggestion
        const hasTransitions = parseInt(transitionDuration) > 0;
```